src/Type/Definition/InterfaceType.php

```diff
@@ -27,6 +30,20 @@ class InterfaceType extends Type implements AbstractType, OutputType, CompositeT
      */
     private $fields;
 
+    /**
+     * Lazily initialized.
+     *
+     * @var InterfaceType[]
+     */
+    private $interfaces;
```

Comment:
Maybe rename to `interfaceList` to differentiate from the map?

Reply:
`ObjectType` follows the convention of having `$interfaces` and `$interfaceMap` so I think consistency would point to keeping `interfaces` for the list type (alternatively, we'd also rename the `$interfaces` variable in ObjectType, which is private).

Reply:
Alright. We don't really suffix with `List` anywhere else, so I think it can stay like this.